### PR TITLE
Add open in Thriveopedia button to organelle icons

### PR DESCRIPTION
**Pull request by Oliveriver**
**Brief Description of What This PR Does**

Adds a button within organelle selection buttons linking to the relevant organelle page in the Thriveopedia.

image

I'm unsure of the design of the button as it adds clutter to the parent button. I thought about scrapping the child button and having the main button link to the Thriveopedia when right-clicked, but that won't be as clear to the player. Open to opinions.

**Related Issues**

N/A

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
I agree about it being a bit cluttered now. I think what could be done is having a specific help button to trigger (F1) about whatever is under the cursor. The tooltip could mention the button to get help about the organelle. Another option I saw recently in Baldur's Gate 3 is that in it you can press "T" to get a tooltip to stick on screen, then you can click on words in the tooltip to get more info about them.

**Comment by ender-titan1** [comment]
I'd propose maybe revealing the help button when hovering over the organelle

**Comment by Oliveriver** [comment]
Just tried showing only on hover and it works ok. I would be happy with it if everyone else is.

Hhyyrylainen's suggestion sounds fine but I'm worried that's not going to be obvious enough to the player, especially right now when they haven't yet had much of a push to visit the Thriveopedia so won't know to do it. Maybe if hover is accompanied by a change in cursor texture to show the key they'd have to press?

**Comment by hhyyrylainen** [comment]
> Hhyyrylainen's suggestion sounds fine but I'm worried that's not going to be obvious enough to the player, especially right now when they haven't yet had much of a push to visit the Thriveopedia so won't know to press that key. Maybe if hover is accompanied by a change in cursor texture to show the key they'd have to press?

That's why I suggested the explicit mention in each tooltip, something like "Press F1 for more info" at the end of all the tooltips. Which I think should have pretty good discoverability. And people who read to the end of the tooltip are likely the people who would be interested in reading more info.

**Comment by ender-titan1** [comment]
If we'd go with hh's suggestion, I'd make the text a slightly different colour so it's differentiated from all the other text

**Comment by Oliveriver** [comment]
How's this? With the note at the bottom of the tooltip below a separator for emphasis.

image

**Comment by hhyyrylainen** [comment]
I think it looks good. Though maybe the tooltip should instead of mentioning Thriveopedia to more prompt the "more info" part? Or maybe a hybrid text like "Press F1 to view more info about this in the Thriveopedia"?

File: src/microbe_stage/editor/tooltips/SelectionMenuToolTip.cs [hunk 1]
```diff
@@ -296,6 +314,16 @@ public void WriteMembraneModifierList(MembraneType referenceMembrane, MembraneTy
         }
     }
 
+    [RunOnKeyDown("help", Priority = int.MaxValue)]
+    public bool OpenMoreInfo()
+    {
+        if (!Visible || thriveopediaPageName == null)
+            return false;
+
+        ThriveopediaManager.OpenPage(Name);
```
Comment (hhyyrylainen) [inline src/microbe_stage/editor/tooltips/SelectionMenuToolTip.cs:323]: Shouldn't this use `thriveopediaPageName` here? I think I'll correct this while fixing the merge conflict to merge this today.

**Review by hhyyrylainen — APPROVED** [review]
I was hoping some text would be found that allows the "open help" tooltip part to fit on a single line, but this is at least good enough for an initial merging of this feature.

**Comment by hhyyrylainen** [comment]
Actually, I'll set the text to this for now as it fits on a single line and kind of still has all the parts of the longer message.

**Pull request merged**